services/tenant-ui/frontend/src/store/governanceStore.ts

```diff
@@ -8,7 +8,7 @@ export const useGovernanceStore = defineStore('governance', () => {
   const schemaTemplates: any = ref(null);
```

Comment:
i guess we need to figure out a standard linter/formatter... this will get annoying...

Reply:
Yeah mine doesn't do this format here when I auto-format... (alt-shift-f) though maybe it _should_ be based on the prettier rules? Not sure but we definitely need a quick look at getting that stuff standardized

Reply:
Agreed. If we can have all our editors read linter rules from the repo .linter files (instead of local env) that would make our lives a lot easier.